arangod/Aql/Ast.cpp

```diff
@@ -1106,6 +1106,24 @@ AstNode* Ast::createNodeValueString(char const* value, size_t length) {
   return node;
 }
 
+AstNode* Ast::createNodeValueString(std::string const &value) {
```

Comment:
This is completely redundant.
Why not simply make it `return createNodeValueString(value.data(), value.size());` instead of repeating an existing function's body?